src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -1138,14 +1130,9 @@ private static string GetUserCodeLocation(Stack stack)
             foreach (StackFrame frame in stack.Frames)
             {
                 string fullyQualifiedLogicalName = frame.Location.LogicalLocation.FullyQualifiedName;
-                Match match = LogicalLocationRegex.Match(fullyQualifiedLogicalName);
```

Comment:
>fullyQualifiedLogicalName [](start = 57, length = 25)

There's no need to strip off the return type here, in this special case, because now we do it on _every_ stack frame when we create it (see below).